wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/id.md

```diff
@@ -40,17 +42,45 @@ Berikut ini adalah ciri-ciri modding yang diharapkan dari Beatmap Nominator. Keg
 `m` = Jumlah minimum yang diharapkan dari beatmap yang di mod secara individu dalam satu bulan\
 `M` = Jumlah total yang dicapai dari beatmap yang di mod secara individu dalam satu bulan
 
-- [Situs web ini](http://bn.mappersguild.com/bnapps) memiliki alat untuk menghitung skor mod di halaman utamanya.
-- Untuk menentukan `Skor Minimum yang Dibutuhkan`, jumlah total Skor Mod dalam tiga bulan terakhir harus lebih besar atau sama dengan nol,
+- [Situs web ini](http://bn.mappersguild.com/bnapps) memiliki alat untuk menghitung `skor mod` di halaman utamanya.
+- Untuk menentukan skor minimum yang dibutuhkan, jumlah total `Skor Mod` dalam tiga bulan terakhir harus lebih besar atau sama dengan nol,
 - Batas minimum keseluruhan ditetapkan berdasarkan nilai berikut:
- - osu!taiko, osu!catch and osu!mania: m = 3 dan total 150 kudosu atau lebih
- - osu!: m = 4 dan total 200 kudosu atau lebih
+ - osu!taiko, osu!catch and osu!mania: `m` = 3 dan total 150 kudosu atau lebih
```

Comment:
```suggestion
 - osu!taiko, osu!catch dan osu!mania: `m` = 3 dan total 150 kudosu atau lebih
```

Sepertinya kata "and" disini ga termasuk kedalam rumus seperti yg ada di query 'AND' , jadi menurutku lebih baik di translate

Reply:
```suggestion
 - osu!taiko, osu!catch, dan osu!mania: `m` = 3 dan total 150 kudosu atau lebih
```